### PR TITLE
Likes Array Of Users in Post and Comment  

### PR DESCRIPTION
**Pull request by shivamarora96**
Users who liked the post can be traced using Likes array in JSON

### CODE REVIEW

**Comment by shivamarora96** [comment]
@tharindupr kindly review this PR :smile:

**Comment by psnmissaka** [comment]
@shivamarora96 currently if the user likes the same post twice it counts as two votes. We have to limit the user from liking a post more than once. We should probably handle that scenario as well

**Comment by shivamarora96** [comment]
@psnmissaka Yeah But this PR would solve that problem

File: stackle_api/app/models/post.js [hunk 1]
```diff
@@ -298,6 +274,89 @@ postSchema.statics.removeTag = function(request, response){
...
+                if(!result)
+                   return returnWithResponse.configureReturnData({ status: 400, success: false, result: `Post-${currentPostId} not found`}, response);
+
+               var likeArray = result.likes;
+               var check = likeArray.indexOf(currentUserId);
+              
+               if(check === -1){
+                    result.likes.push(currentUserId);
+                    result.save();
+                    return returnWithResponse.configureReturnData({ status: 200, success: true, result: `${currentUserId}: user liked the post-${currentPostId}`}, response);
```
Comment (shivamarora96) [inline stackle_api/app/models/post.js]: @psnmissaka this would solve the double liking issue

Comment (psnmissaka) [reply stackle_api/app/models/post.js]: @ntbandara3 update the front-end accordingly. If someone has liked a post and presses the like button again then trigger the unlike.

**Comment by shivamarora96** [comment]
@psnmissaka changes Made. Kindly review it

File: stackle_api/app/models/post.js (hunk 1 above)
Comment (ntbandara3) [reply stackle_api/app/models/post.js]: @psnmissaka sure. working on it.

**Pull request merged**